### DISCUSSION TITLE
Here's what to do while you're waiting!

### DISCUSSION THREAD

**Discussion by Meta0Music** [3kh0/ext-remover#127] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
OK

So seeing a lot of the posts here I can see you all are absolutely itching for your games and socials back, but as of right now things have collapsed behind the scenes. It is very much being worked on, but it can't be done overnight. So here are some other things you can do :D

### **DEPLOY 3kh0.github.io**

The website that 3kh0 has made will probably be more than enough for you to atleast keep yourself busy. Check it out by going to their repositories, clicking on "3kh0.github.io" and scrolling down till you see all the deploying buttons. Be sure to star the repository to support them!

### **USE PROXIES**

Some people are really ignoring this, which is sad. Use proxies to your advantage, there are plenty of them, and some of them have tab cloaking abilities, but please just use those in the meantime instead of complaining about them.

If you want to drop some proxy links, leave them below :3

I'll be here if you need any help :D

**Comment by ghost** [comment, marked as answer]
You could also use `javascript:(function(){window.onbeforeunload=function(){return '';}})();`. How you do it: press search+esc, crash goguardian, get ready to quickly visit the site. Keep goguardian crashed, visit the site, spam the bookmarklet then just leave it until it asks if you want to leave. press cancel. this will happen every so often, press cancel every time. some sites won't let you redirect to anythiing, but if you're on youtube or something, don't worry just press cancel when you visit a video or look at it in miniplayer
